template.tpl

```diff
@@ -32,83 +32,227 @@ ___INFO___
-    "type": "TEXT"
-  },
-  {
-    "displayName": "Instance Name",
-    "simpleValueType": true,
-    "name": "instanceName",
-    "type": "TEXT",
-    "valueHint": "Default (blank)",
-    "help": "If you want to run more than one Amplitude instance (with different API keys or different initialization options), you can provide an \u003cstrong\u003eInstance Name\u003c/strong\u003e here. Make sure that all the Amplitude tags that should reference this instance have the Instance Name configured in the tag settings."
-  },
   {
     "selectItems": [
       {
-        "displayValue": "init",
+        "displayValue": "Initialize (init)",
         "value": "init"
       },
       {
-        "displayValue": "track",
+        "displayValue": "Track Event (track)",
         "value": "track"
       },
       {
-        "displayValue": "identify",
+        "displayValue": "Set User Properties (identify)",
         "value": "identify"
       },
       {
-        "displayValue": "setGroup",
+        "displayValue": "Set User ID (setUserId)",
+        "value": "setUserId"
+      },
+      {
+        "displayValue": "Set Group (setGroup)",
         "value": "setGroup"
       },
       {
-        "displayValue": "groupIdentify",
+        "displayValue": "Set Group Properties (groupIdentify)",
         "value": "groupIdentify"
       },
       {
-        "displayValue": "revenue",
+        "displayValue": "Track Revenue (revenue)",
         "value": "revenue"
       },
       {
-        "displayValue": "flush",
+        "displayValue": "Flush Events (flush)",
         "value": "flush"
       },
       {
-        "displayValue": "setUserId",
-        "value": "setUserId"
-      },
-      {
-        "displayValue": "setDeviceId",
+        "displayValue": "Set Device ID (setDeviceId)",
         "value": "setDeviceId"
       },
       {
-        "displayValue": "setSessionId",
+        "displayValue": "Set Session ID (setSessionId)",
         "value": "setSessionId"
       },
       {
-        "displayValue": "reset",
+        "displayValue": "Reset User (reset)",
         "value": "reset"
       },
       {
-        "displayValue": "setOptOut",
+        "displayValue": "Set Opt-Out (setOptOut)",
         "value": "setOptOut"
       }
     ],
     "displayName": "Tag Type",
     "defaultValue": "init",
     "simpleValueType": true,
     "name": "type",
-    "type": "SELECT"
+    "type": "SELECT",
+    "alwaysInSummary": true
+  },
+  {
+    "type": "LABEL",
+    "name": "initDescription",
+    "displayName": "\u003ca href\u003d\"https://www.docs.developers.amplitude.com/data/sources/google-tag-manager-client/#init\"\u003einit\u003c/a\u003e - This tag sets up Amplitude on the page. You likely only need one init tag, and it should be triggered on All Pages or a similar trigger that fires consistently on each page load. See also \u003ca href\u003d\"https://www.docs.developers.amplitude.com/data/sdks/browser-2/#initialize-the-sdk\"\u003eSDK Reference\u003c/a\u003e",
```

Comment:
I think we should just refer to "init" as "Initialize" although out the template, but leave it like "Initialize (init)" only in the tag type dropdown because customers might be accustomed to that already. I believe the "full words" can relate better to non-technical user and benefit activation

Reply:
Suggestion:

> You only need one Initialize tag and it should be triggered on All Pages or a similar trigger that fires consistently on each page load